src/lib/operators/multi_predicate_join/multi_predicate_join_evaluator.cpp

```diff
@@ -56,6 +57,28 @@ bool MultiPredicateJoinEvaluator::fulfills_all_predicates(const RowID& left_row_
   return true;
 }
 
+PredicateEvaluationResult MultiPredicateJoinEvaluator::satisfies_all_predicates_null_exit(const RowID& left_row_id,
```

Comment:
Ah ... now I see a differentiation but the naming does not tell me, what's the difference between both methods.

Reply:
Both methods have a very similar name but different return types. I think that's unnecessary confusing as well.